go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/Masterminds/sprig/v3 v3.3.0
 	github.com/adrg/xdg v0.5.3
 	github.com/blang/semver/v4 v4.0.0
-	github.com/coredns/corefile-migration v1.0.24
```

Comment:
Let's also update the docs referring to CoreDNS versions (i.e. which version which CAPI version supports)